fmpz/divisible_si.c

```diff
@@ -14,22 +14,23 @@
 #include "ulong_extras.h"
 #include "fmpz.h"
 
-int fmpz_divisible_si(const fmpz_t x, slong p)
+int
+fmpz_divisible_si(const fmpz_t f, slong g)
```

Comment:
This will need new test code which exercises it for the new inputs that are supported.

Reply:
Done.